bin/functions

```diff
@@ -15,13 +15,13 @@ case "${KERNEL_NAME}" in
     Darwin) BASE64_OPTS="-D"
             OS="mac"
             SHA256SUM="gsha256sum"
-            STAT="/usr/bin/stat -f %c ${CACHE_DIR}/*"
+            STAT_OPTS="-f %c"
```

Comment:
```suggestion
            STAT_OPTS=('-f' '%c')
```
Use array instead of string, otherwise this will be passed as 1 argument instead of 2 into the `stat` command. It actually also sort of works because the space seems to be optional between `-f` and its format, thus the resulting format was actually " %c" giving an extra space to the resulting epoch which probably will be removed when used for arithmetic comparsion in subsequent steps. But changing it to an array has the advantage of being correct in case more arguments are added on subsequently.

Reply:
I agree with you on using an array for these.